lib/common.sh

```diff
@@ -579,14 +695,8 @@ if [ ! -f $LOGFILE ]; then
     fi
 fi
 
-if [ ! -f $MOUNTMAP ]; then
-    touch $MOUNTMAP
-    if [ $? -ne 0 ]; then
-        eecho "[FATAL] Not able to create '${MOUNTMAP}'!"
-        exit 1
-    fi
-    chattr -f +i $MOUNTMAP
-fi
+# Create mount map file
+create_mountmap_file
```

Comment:
it's better not to call exit from inside a common function, rather return 1 from there and do

if ! create_mountmap_file; then
exit 1
fi

this is clearer to read that we are exiting if we fail to create the file